keck_vnc_launcher.py

```diff
@@ -824,6 +863,79 @@ def get_vncviewer_properties(self):
             self.vncviewer_has_geometry = False
 
 
+    def get_api_data(self, api_key, account):
+        '''Get data from API which contains all info needed to connect.'''
+        self.api_data = None
+
+        #form API url and get data
+        url = f'{KRO_API}?key={self.api_key}'
+        if account is not False: url += f'&account={self.args.account}'
+        self.log.debug(f'Calling API with URL: {url}')
```

Comment:
We should also log what responses we got from the API.  If we don't want to put some info in the log for security reasons, we should just have a log line which says that we got that info.

Reply:
ok done.  Logging API return code and if successfully parsed.  I'm thinking we shouldn't log the actual response b/c of all the sensitive info in it.  Easy to add if we change mind.